packages/ui/useScroll/Scrollbar.js

```diff
@@ -33,87 +33,110 @@ const setCSSVariable = (element, variable, value) => {
 };
 
 /**
- * An unstyled base component for a scroll bar.
+ * A custom hook that passes scrollbar behavior information as its render prop.
  *
- * @function ScrollbarBase
+ * @class
```

Comment:
why it is class?

Reply:
for documentation, let's keep it now.

Reply:
I added this to refer this as ui/useScroll.useScrollbar since there is no format for hooks. I think we could revisit all this type later.